pkg/profiles/default.go

```diff
@@ -0,0 +1,69 @@
+package profiles
+
+import apiv1 "github.com/acorn-io/runtime/pkg/apis/api.acorn.io/v1"
+
+// Default values
+var (
+	ClusterDomainDefault         = ".local.oss-acorn.io"
+	InternalClusterDomainDefault = "svc.cluster.local"
+
+	AcornDNSEndpointDefault = "https://oss-dns.acrn.io/v1"
+	AcornDNSStateDefault    = "auto"
+
+	// LetsEncryptOptionDefault is the default state for the Let's Encrypt integration
+	LetsEncryptOptionDefault = "disabled"
+
+	// DefaultImageCheckIntervalDefault is the default value for the DefaultImageCheckInterval field
+	DefaultImageCheckIntervalDefault = "5m"
```

Comment:
nit: this seems like a weird name to me

```suggestion
	// AutoUpgradeIntervalDefault is the default value for the AutoUpgradeInterval field
	AutoUpgradeIntervalDefault = "5m"
```

(would also need to update this in other places in this file)